### ISSUE TITLE
(PS3) Overscan issues on GLUI/MaterialUI do this driver menu unusable

### ISSUE THREAD

**Issue by Ezio-PS** [libretro/RetroArch#3255]
On RA 1.3.6 for PS3 there're overscan issues when the driver menu is set on GLUI/MaterialUI. It becomes almost unusable, the menu entries aren't easily readable and it's really difficult to be able to use RA with this driver menu.

**Comment by fr500** [comment]
is this still an issue?

**Comment by Ezio-PS** [comment]
Yes and not. I found a workaround some time ago setting the dpi to 500 https://github.com/libretro/RetroArch/pull/3300/commits/e45a496b07a95b74842532d36343db8d3254faad
But it's not a proper fix, though the GLUI menu driver is now at least usable. 
In addition, i noticed that the overscan issue is concerning all the menu drivers and the in-game experience.
@Kivutar and @twinaphex know well the issue, maybe borders and some other stuff need to be done to definitively fix the issue.

**Comment by Kivutar** [comment]
We could add a new video setting that adds black borders around the rendered frame.

I know that there already is the custom viewport for this, but the custom viewport only affects the game windows. And we can't have the menus using the same viewport as the game, (how would the menu look on a vertical arcade game?)

I don't know where is the best place to implement it. But I guess it would be at the context driver level or in the graphic driver.
